tools/nightly_build_and_test_stats.groovy

```diff
@@ -666,23 +788,59 @@ node('worker') {
                     }
                 } else {
                     // Check if build in-progress
-                    inProgressBuildUrl = getInProgressBuildUrl(trssUrl, variant, featureRelease, status['expectedReleaseName'].replaceAll("-beta", ""), status['upstreamTag']+"_adopt")
+                    (probableBuildUrl, probableBuildIdForTRSS, probableBuildStatus) = getBuildUrl(trssUrl, variant, featureRelease, status['expectedReleaseName'].replaceAll("-beta", ""), status['upstreamTag']+"_adopt")
```

Comment:
This "getInProgressBuildUrl" was originally added to purely see if A build was in-progress for the given tag, regardless of what platform, is was just a "rough" status as to "is there anything still going...?"
Thus it's intention was not for finding a "specific" build, which is what I think we need now, the situation can be that in TRSS there could be several entries for a given tag say "jdk-21.0.5+9-ea", which has #334 and #335, as Windows_aarch64 was re-run... As it stands the above will find #335(WindowsAarch64) and incorrectly think no ReproducibleTest ran for x64Linux, which is in #334.

We could update verifyReleaseContent() to add the list of platforms to the status[] json (eg.status['foundAssets']), and update getBuildUrl to take a build platform, and find the TRSS build entry for the given tag that has that build platform...?

Reply:
After in-person discussion, me and Andrew have agreed that per-platform pipeline identification is the optimum way to produce reproducibility percentages in the long run.

However, in the spirit of getting an MVP merges asap, we will be merging the code as-is, and breaking out the per-platform solution into a separate issue (link shortly).

Reply:
https://github.com/adoptium/ci-jenkins-pipelines/issues/1132